code/game/objects/items/binoculars.dm

```diff
@@ -34,6 +34,7 @@
 /obj/item/binoculars/tactical/examine()
 	..()
 	to_chat(usr, "<span class='notice'>They are currently set to [mode ? "range finder" : "CAS marking"] mode.</span>")
+	to_chat(usr, "<span class='notice'>The internal GPS reads: [src.x][src.y].</span>")
```

Comment:
Put it on the same `to_chat` as the previous (use `<br>` if needed), and remove the `src` references.